test/sql/cast/string_to_struct_cast.test_slow

```diff
@@ -0,0 +1,546 @@
+# name: test/sql/cast/string_to_struct_cast.test_slow
+# description: cast strings into (nested) structs
+# group: [cast]
+
+statement ok
+PRAGMA enable_verification
```

Comment:
Could we split this into a set of fast tests and a set of slow tests? The slow tests are not run in debug mode, and hence certain types of errors might be missed in the CI.